docs/howto/error-reporting.txt

```diff
@@ -255,13 +255,6 @@ following methods:
     Returns ``True`` to activate the filtering operated in the other methods.
     By default the filter is active if :setting:`DEBUG` is ``False``.
 
-.. method:: SafeExceptionReporterFilter.get_request_repr(request)
```

Comment:
I'm not following your argument. It seems you are saying that because data could leak elsewhere we shouldn't try to sanitize anywhere? Are you opposed to the idea if we make it a separate ticket?